storage/engine/rocksdb.go

```diff
@@ -55,6 +55,13 @@ func NewRocksDB(attrs proto.Attributes, dir string, cacheSize int64) *RocksDB {
 	}
 }
 
+func newMemRocksDB(attrs proto.Attributes, cacheSize int64) *RocksDB {
+	return &RocksDB{
```

Comment:
what about this RocksDB instance makes it inMem? From the above, I would've guessed that you just use the inMemAttrs instead of passing in an `attrs` parameter.